src/SFA.DAS.RoATPService.Application.Api/StartupConfiguration/Startup.cs

```diff
@@ -0,0 +1,180 @@
+
+                services.AddSwaggerGen(c =>
+                {
+                    c.SwaggerDoc("v1", new Info { Title = "SFA.DAS.RoATPService.Application.Api", Version = "v1" });
+
+                    if (_env.IsDevelopment())
+                    {
+                        var basePath = AppContext.BaseDirectory;
+                        var xmlPath = Path.Combine(basePath, "SFA.DAS.RoATPService.Application.Api.xml");
+                        c.IncludeXmlComments(xmlPath);
+                    }
+                });
+
+                serviceProvider = ConfigureIOC(services);
+
+                if (_env.IsDevelopment())
+                {
+                   // TestDataService.AddTestData(serviceProvider.GetService<AssessorDbContext>());
+                }
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, "Error during Startup Configure Services");
+                throw;
+            }
+
+            return serviceProvider;
+        }
+
+        private IServiceProvider ConfigureIOC(IServiceCollection services)
+        {
+            var container = new Container();
+
+            container.Configure(config =>
+            {
+                config.Scan(_ =>
+                {
+                    _.AssembliesFromApplicationBaseDirectory(c => c.FullName.StartsWith("SFA"));
+                    _.WithDefaultConventions();
+
+                    _.ConnectImplementationsToTypesClosing(typeof(IRequestHandler<>)); // Handlers with no response
+                    _.ConnectImplementationsToTypesClosing(typeof(IRequestHandler<,>)); // Handlers with a response
+                    _.ConnectImplementationsToTypesClosing(typeof(INotificationHandler<>));
+                });
+
+                config.For<IWebConfiguration>().Use(Configuration);
+                config.For<SingleInstanceFactory>().Use<SingleInstanceFactory>(ctx => t => ctx.GetInstance(t));
+                config.For<MultiInstanceFactory>().Use<MultiInstanceFactory>(ctx => t => ctx.GetAllInstances(t));
+                config.For<IMediator>().Use<Mediator>();
+                config.For<IDbConnection>().Use(c => new SqlConnection(Configuration.SqlConnectionString));
+
+                config.Populate(services);
+            });
+
+            return container.GetInstance<IServiceProvider>();
+        }
+
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            try
+            {
+
+                MappingStartup.AddMappings();
+
+                if (env.IsDevelopment()) app.UseDeveloperExceptionPage();
+
+                app.UseSwagger()
+                    .UseSwaggerUI(c =>
+                    {
+                        c.SwaggerEndpoint("/swagger/v1/swagger.json", "SFA.DAS.RoATPService.Application.Api v1");
+                    });
+                //    .UseAuthentication();
+
```

Comment:
Why is this commented out? Can you check you can't leave it in?

Reply:
Good spot, commented out in error